packages/fluentui/react-bindings/tsconfig.json

```diff
@@ -25,6 +25,9 @@
     },
     {
       "path": "../react-component-ref"
+    },
+    {
+      "path": "../react-northstar-fela-renderer"
```

Comment:
Just a question - do we have/can we add any way to enforce project references to be up-to-date?

Reply:
I don't know about it, but if there will be a missing reference it will throw.